internal/releases/team.go

```diff
@@ -169,9 +169,9 @@ func (t *Team) populateGithubRepos(org config.GithubOrg) error {
 		}
 		wg.Wait()
 
-		// Iterate over repos, add only those that have releases to the Team's list of repos
+		// Iterate over repos and add the unarchived ones that have at least one commit
 		for _, r := range repos {
-			if len(r.Info().Releases) > 0 {
+			if !r.Info().IsArchived && len(r.Info().Releases) > 0 {
```

Comment:
This change will be needed if we break Commits/Releases into separate fields.

```suggestion
			if !r.Info().IsArchived && (len(r.Info().Releases) > 0 || len(r.Info.Commits) > 0) {
```